packages/jaspr_riverpod/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased 0.3.8
```

Comment:
Make it `Unreleased patch`. The pipeline will do the version bump.

Reply:
@schultek Thanks, fixed!